upcloud/load_balancer.go

```diff
@@ -167,8 +168,9 @@ type LoadBalancerFrontendTLSConfig struct {
 
 // LoadBalancerFrontendProperties represents frontend properties
 type LoadBalancerFrontendProperties struct {
-	TimeoutClient        int  `json:"timeout_client,omitempty"`
-	InboundProxyProtocol bool `json:"inbound_proxy_protocol"`
+	TimeoutClient        int   `json:"timeout_client,omitempty"`
+	InboundProxyProtocol *bool `json:"inbound_proxy_protocol,omitempty"`
```

Comment:
This change would introduce a breaking change. I'd see it useful for consistency with `HTTP2Enabled` and backend properties being boolean pointers, but can be reverted if needed.

Reply:
This and `HTTP2Enabled` could be just `bool` with `omitempty`, and that would not be a breaking change anymore, unless clients are relying on logic that not setting a value would result in `false` in a modification request for example.

Reply:
I'm not totally against it, but we could also try to remember do this change next time we have also other breaking changes in queue.

Reply:
Reverted and added a task for changing this and similar fields for a major release.